price-feeder/oracle/provider/binance.go

```diff
@@ -69,44 +72,49 @@ func (p BinanceProvider) GetTickerPrices(tickers ...string) (map[string]sdk.Dec,
 	return tickerPrices, nil
 }
 
-func (p BinanceProvider) getTickerPrice(ticker string) (sdk.Dec, error) {
-	path := fmt.Sprintf("%s/api/v3/ticker/price?symbol=%s", p.baseURL, ticker)
+func (p BinanceProvider) getTickerPrice(ticker string) (TickerPrice, error) {
+	path := fmt.Sprintf("%s/api/v3/ticker/24hr?symbol=%s", p.baseURL, ticker)
```

Comment:
Endpoint should probably be in the BinanceProvider struct

Reply:
Ehh I don't think there's must merit to adding it to the type, but we can define a const instead 👍